draft-ietf-tls-dtls13.md

```diff
@@ -956,6 +957,10 @@ configured not to perform a cookie exchange.  The default SHOULD be
 that the exchange is performed, however.  In addition, the server MAY
 choose not to do a cookie exchange when a session is resumed or, more
 generically, when the DTLS handshake uses a PSK-based key exchange.
+Servers which process 0-RTT requests and send 0.5-RTT responses
+without a cookie exchange should
+store the client's IP in the cookie and limit the size of their
```

Comment:
IMPORTANT: you just said that no cookie exchange is involved.  I think that you mean session ticket.

```suggestion
store the client's IP in the session ticket and limit the size of their
```

Reply:
yeah, clearly it's getting too late in the day for me to edit.